src/components/Patient/ManagePatients.tsx

```diff
@@ -991,23 +986,59 @@ export const PatientManager = () => {
         }}
       />
 
-      <div className="mt-4 gap-4 lg:gap-16 flex flex-col lg:flex-row lg:items-center">
-        <div id="total-patientcount">
-          <CountBlock
-            text={t("total_patients")}
-            count={data?.count || 0}
-            loading={isLoading}
-            icon="d-patient"
-          />
+      <div className="manualGrid my-4 mb-[-12px] mt-5 grid-cols-1 gap-3 px-2 sm:grid-cols-4 md:px-0">
+        <div className="mt-2 flex h-full flex-col gap-3 xl:flex-row">
+          <div className="flex-1" id="total-patientcount">
+            <CountBlock
+              text="Total Patients"
+              count={data?.count || 0}
+              loading={isLoading}
+              icon="l-user-injured"
+              className="pb-12"
+            />
+          </div>
+        </div>
+        <div className="col-span-3 w-full">
+          <div className="mt-2">
+            <div className="mb-4 mt-1 md:flex md:gap-4">
+              <SearchInput
+                label="Search by Patient"
+                placeholder="Enter patient name"
+                {...queryField("name")}
+                className="w-full grow"
+              />
+              <SearchInput
+                label="Search by IP/OP Number"
+                placeholder="Enter IP/OP Number"
+                secondary
+                {...queryField("patient_no")}
+                className="w-full grow"
+              />
+            </div>
+            <div className="mb-4 md:flex md:gap-4">
+              <PhoneNumberFormField
+                label="Search by Primary Number"
+                {...queryField("phone_number", "+91")}
+                value={phoneNumber}
+                onChange={(e) => setPhoneNumber(e.value)}
+                types={["mobile", "landline"]}
+                className="w-full grow"
+                error={((phoneNumber || "+91") === "+91" && "") || undefined}
+              />
+              <PhoneNumberFormField
+                label="Search by Emergency Number"
+                {...queryField("emergency_phone_number", "+91")}
+                value={emergencyPhoneNumber}
+                onChange={(e) => setEmergencyPhoneNumber(e.value)}
+                types={["mobile", "landline"]}
+                className="w-full"
+                error={
+                  ((emergencyPhoneNumber || "+91") === "+91" && "") || undefined
+                }
+              />
+            </div>
+          </div>
         </div>
-
-        <SearchByMultipleFields
```

Comment:
Merge error here; SearchByMultipleFields should be included instead of the individual search fields.